src/js/models/panel_model.js

```diff
@@ -478,7 +479,9 @@
                 scale = Math.max(xPercent, yPercent);
 
             // if not zoomed or panned and panel shape is approx same as image...
-            if (dx === 0 && dy === 0 && zoom == 100 && Math.abs(xPercent - yPercent) < 0.01) {
+            var orig_wh = orig_width / orig_height,
+                view_wh = width / height;
```

Comment:
could height be 0?

Reply:
See above - ensuring that users don't create panels with zero width or height makes the most sense, but should probably go in a separate PR. There's plenty of places that would break in that case, so this PR doesn't add any new bugs for that.